docs-chef-io/content/inspec/install.md

```diff
@@ -96,10 +83,275 @@ For SUSE Linux Enterprise Server, use the following command to install Chef InSp
+       Release:   22.5.0
+       Arch:      arm64
+
+   inspec> exit
+   ```
+
+Chef InSpec validates the license key, displays information about the license entitlements, and then runs `inspec shell` as requested.
+Chef InSpec stores license keys for future use, and InSpec will not prompt you for the license key for the duration of your license.
+
+#### Command line option
+
+You can set the license key in the command line using the `--chef-license-key` option.
+You may provide this argument to most Chef InSpec CLI main commands, however some plugins may not support the flag.
+
+```bash
+inspec exec profile_name --chef-license-key <LICENSE_KEY>
+```
+
+#### Environment variable
+
+You can set the license key as an environment variable using the `CHEF_LICENSE_KEY`
+Chef InSpec will read the license key from the variable and attempt to validate the key.
+If successful, InSpec saves the key and will not prompt you for it the next time you run InSpec.
+
+```bash
+export CHEF_LICENSE_KEY=<LICENSE_KEY>
+inspec exec profile_name
+```
+
+#### Chef Private License Server
+
+For large or isolated (air-gapped) fleets, Chef InSpec can retrieve a license key from a Chef Private License Server.
+With a Chef Private License Server, InSpec users do not need to know a license key -- only the URL(s) of the server.
+
+Chef InSpec sends a request to the Private License Server for a list of license keys and then uses that response to license itself during execution.
+InSpec will not prompt you for a license key.
+Chef InSpec does not store license keys for long-term use when they are retrieved from a Chef Private License Server.
+
+You can set the URL of a Chef Private License Server using an [environment variable](#environment-variable-1) or with a [command line option](#command-line-option-1).
+
+For details about how to obtain, setup, and run a Chef Private License Server, please contact LINK TODO.
+
+##### Environment variable
+
+Use the `CHEF_LICENSE_SERVER` environment variable to set a Chef Private License Server URL.
+
+```bash
+export CHEF_LICENSE_SERVER=https://license-server.example.com
+inspec exec profile_name
+```
+
+##### Command line option
+
+Use the `--chef-license-server` command line option to set a Chef Private License Server URL.
+
+```bash
+inspec exec profile_name --chef-license-server https://license-server.example.com
+```
+
+##### Multiple license servers
+
+You can set multiple Chef Private License Servers which provides resiliency and redundancy for managing licenses.
+
+Enter up to five Chef Private License Server URLS as a comma-separated list. Chef InSpec will try each URL and use the first one that works.
+
+```bash
+export CHEF_LICENSE_SERVER=https://license-server-01.example.com,https://license-server-02.example.com
+inspec exec profile_name
+```
+
+This capability is basic and you must synchronize the license servers, otherwise you may get inconsistent results.
+
+### Generate a license key
```

Comment:
This section needs to be updated / replaced based on the upcoming changes to replaces the CLI generation with a web form.